mapclientplugins/filelocationsinkstep/ui_configuredialog.py

```diff
@@ -96,9 +96,9 @@ def setupUi(self, ConfigureDialog):
     def retranslateUi(self, ConfigureDialog):
         ConfigureDialog.setWindowTitle(QCoreApplication.translate("ConfigureDialog", u"Configure File Location Sink", None))
         self.configGroupBox.setTitle("")
-        self.label1.setText(QCoreApplication.translate("ConfigureDialog", u"Output directory:  ", None))
-        self.label2.setText(QCoreApplication.translate("ConfigureDialog", u"prefix", None))
-        self.label0.setText(QCoreApplication.translate("ConfigureDialog", u"identifier:  ", None))
+        self.label2.setText(QCoreApplication.translate("ConfigureDialog", u"output directory:", None))
+        self.label1.setText(QCoreApplication.translate("ConfigureDialog", u"prefix:", None))
+        self.label0.setText(QCoreApplication.translate("ConfigureDialog", u"identifier:", None))
```

Comment:
Make all of these start with a capital letter and end with a space after the colon.